app/grandchallenge/components/templates/components/componentinterface_list.html

```diff
@@ -67,6 +68,10 @@ <h2>{{ object_type|title }} {{ list_type|title }} Options</h2>
                                 <code>/input/{{ interface.relative_path }}{% if interface.relative_path %}/{% endif %}&lt;uuid&gt;.tif</code>
                             {% else %}
                                 <code>/input/{{ interface.relative_path }}</code>
+                                {% if interface.example_value %}
+                                    <br>
+                                    <a download="json_schema_example.json" href="data:application/octet-stream;charset=utf-8;base64,{{ interface.example_value | b64encode_json }}">Download schema example</a>
```

Comment:
Downloading a file to see the content is awkward. It is better to display the value. If the value does not fit into the table then put it in a modal.

Reply:
I made it view in modal instead

Reply:
The title of that column and the objects are still not correct. You are confusing concepts here. This is an example value. It still has nothing to do with schemas.

Reply:
Got it. A quick discussion with Chris helped a lot clarifying the confusion. I fixed it.